### DISCUSSION TITLE
I am aware of Android limitation for Web extention but it seems that it is changing for Firefox Mobile / Fennec?

### DISCUSSION THREAD

**Discussion by 2Belette** [floccusaddon/floccus#1434] [Q&A] [answered]
https://blog.mozilla.org/addons/2023/08/10/prepare-your-firefox-desktop-extension-for-the-upcoming-android-release/
https://www.androidpolice.com/firefox-bringing-back-full-extension-support-android/

Would it make any change to support Fennec / Firefox for Android with Floccus ? That's would bring the gap of not being able to sync the tabs/bookmarks from mobile browser (obviously the reverse is working through the Floccus app)

(from Fennec, I tried to install the Web Extension using the browser as 'Desktop Site', it is installed and I am able to go through the process to add a New Webdav Account but the latest stage (4/4) nothing happen when I try to click on the latest 'Continue', probably limitation on permission or so.

Thanks

**Comment by marcelklehr** [comment, marked as answer]
Yes, once Mozilla implement the necessary bookmarks APIs that floccus needs, it should be straightforward to get floccus working on Firefox for android. The earliest release of this new extension support rollout will not support the bookmarks APIs yet, though, I asked them.
